### PR TITLE
Add statsd receiver

### PR DESCRIPTION
**Pull request by VineethReddy02**
[Statsd](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/statsdreceiver) receiver 
As part of #128

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,6 +85,8 @@ For a general overview of the directories from this operator and what to expect
 
 Your contribution is welcome! For it to be accepted, we have a few standards that must be followed.
 
+If you are contributing to sync the receivers from [otel-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib). Operator only syncs receivers that aren't scrapers. As operator has nothing to do with scrapers. whereas with general receivers, operator exposes the port in kubernetes service object as specified in the receiver's config.
```
Comment (jpkrohling) [inline CONTRIBUTING.md:88]:
```suggestion
If you are contributing to sync the receivers from [otel-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib), note that the operator only synchronizes receivers that aren't scrapers, as there's no need to open ports in services for this case. In general, receivers would open a UDP/TCP port and the operator should be adding an entry in the Kubernetes Service resource accordingly.
```

**Review by jpkrohling — COMMENTED** [review]
LGTM, just changed the wording on the contributing guide a bit.

**Comment by jpkrohling** [comment]
@VineethReddy02 what's the status of this one?

**Comment by VineethReddy02** [comment]
I will update the suggested change and will rebase the PR. :)

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**